submodules/wizard/wizard.js

```diff
@@ -1807,6 +1822,13 @@ define(function(require) {
 					});
 				},
 				function(limits, waterfallCallback) {
+					if (_.chain(limits).pick(_.keys(newLimits)).isEqual(newLimits).value()) {
+						// New limits are equal to the default ones,
+						// so there is no need for update
+						waterfallCallback(null, limits);
```

Comment:
It is always good practice to `return callback(err, [result])` whenever a callback call is not the last statement of a function:
```suggestion
						return waterfallCallback(null, limits);
```